<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Naveen-Mac.local" ignored="1" name="Go Rest regression Test" tests="9" failures="1" timestamp="29 Aug 2019 03:41:34 GMT" time="3.537" errors="0">
  <testcase name="createUserAPIPOSTTest" time="1.247" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="1.365" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="1.475" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="1.603" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="1.696" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="createUserAPIPOSTTest" time="1.759" classname="com.qa.gorest.tests.CreateUserAPITest"/>
  <testcase name="getUserListAPITest" time="2.635" classname="com.qa.gorest.tests.GetUserAPITest"/>
  <testcase name="deleteUserTest" time="2.754" classname="com.qa.gorest.tests.DeleteUserAPITest"/>
  <testcase name="createUserAPIPUTTest_WithSetter" time="1.031" classname="com.qa.gorest.tests.UpdateUserAPITest">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at io.restassured.path.json.JsonPath.getString(JsonPath.java:352)
at com.qa.gorest.tests.UpdateUserAPITest.createUserAPIPUTTest_WithSetter(UpdateUserAPITest.java:82)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 18 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- createUserAPIPUTTest_WithSetter -->
  <testcase name="createUserAPIPUTTest_WithConstructor" time="0.0" classname="com.qa.gorest.tests.UpdateUserAPITest">
    <ignored/>
  </testcase> <!-- createUserAPIPUTTest_WithConstructor -->
</testsuite> <!-- Go Rest regression Test -->
